default.nix

```
@@ -19,13 +19,18 @@ pog.pog {
       description = "The prefix to use in conjunction with system string when pinning paths";
       argument = "PREFIX";
     }
+    {
+      name = "derivations";
```

```suggestion
      name = "name";
```

Since these refer to names of the derivations.